packages/popup-component/src/components/popup/popup.scss

```diff
@@ -0,0 +1,97 @@
+@import "shared.scss";
+
+.close {
+  color: #000;
+  cursor: pointer;
+  float: right;
+  font-size: 2.25rem;
+  font-weight: 600;
+  line-height: 1;
+  opacity: 0.5;
+  text-shadow: 0 1px 0 #fff;
+
+  &:hover {
+    color: #000;
+    text-decoration: none;
+  }
+
+  &:not(:disabled):not(.disabled):hover,
+  &:not(:disabled):not(.disabled):focus {
+    opacity: 0.75;
+  }
+}
+
+.notification {
+  color: $gray;
+  font-family: "Libre Franklin", sans-serif;
```

Comment:
If we have bootstrap variable you probably have `$font-family-sans-serif`

Reply:
We don’t have bootstrap or bootstrap variables here